pr_check.sh

```diff
@@ -2,10 +2,25 @@
 
 set -exvo pipefail -o nounset
 
-IMAGE_TEST=addon-metadata-operator
+function changes_include(){
+     changed_files=$(git diff origin/master..HEAD --name-only | grep -c "$1")
 
-docker build -t ${IMAGE_TEST} -f Dockerfile.ci .
-docker run --rm ${IMAGE_TEST} check test test-e2e build
+     [ "${changed_files}" -gt 0 ]
+}
 
-# build addon-metadata-operator containers
-make docker-build
+GO_1_16="/opt/go/1.16.15/bin"
+
+if [ -d  "${GO_1_16}" ]; then
+     PATH="${GO_1_16}:${PATH}"
+fi
+
+# pre-emptively install go-sqlite3 to ensure almagamated libsqlite3
```

Comment:
typo almagamated -> amalgamated. I was checking the meaning of it :)

Reply:
Fixed